iree/hal/device.h

```diff
@@ -189,9 +189,25 @@ IREE_API_EXPORT iree_status_t IREE_API_CALL iree_hal_device_queue_submit(
     iree_hal_queue_affinity_t queue_affinity, iree_host_size_t batch_count,
     const iree_hal_submission_batch_t* batches);
 
+// Submits batches of work and waits |wait_semaphore| reaches |wait_value|.
```

Comment:
"waits _until_ |wait_semaphore| reaches..."

```suggestion
// Submits batches of work and waits until |wait_semaphore| reaches
// |wait_value|.
```

(or reword so this all fits on one line again)